mitodl/mitxpro
# 140 Product Page Header

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #140 

#### What's this PR do?
Adds the header (#140) to course detail page. Also adds data fields for marketing, managed through the CMS.

#### How should this be manually tested?
Set up a Page for a Course through the CMS. Verify the header is as per design and all configured fields are showing up correctly.

#### Screenshots (if appropriate)
![header-desktop-screeny](https://user-images.githubusercontent.com/45350418/56410617-201f0b80-6297-11e9-83aa-ed096e06e17e.png)

![header-mobile-screeny](https://user-images.githubusercontent.com/45350418/56410923-5c9f3700-6298-11e9-8f4a-43195c1d0d33.png)

![wagtail-edit-course-screeny](https://user-images.githubusercontent.com/45350418/56427558-c68a0180-62d5-11e9-8b14-67996bf98d7b.png)
